docker-compose.yml

```diff
@@ -442,7 +442,7 @@ services:
     environment:
       ERRBIT_API_KEY: 1
       ERRBIT_ENVIRONMENT_NAME: draft-government-frontend
-      GOVUK_APP_NAME: draft-government-frontend
+      GOVUK_APP_NAME: government-frontend
```

Comment:
I notice this introduces an inconsistency as all the other apps on the draft stack in this repo are prefixed with draft.

I've just been trying to think if this is fine to do or not, and I think it will potentially introduce problems as it's used as a namespace in applications (e.g. sidekiq jobs). I imagine it doesn't prove a problem in production though as the servers are isolated.

Might be a less risky to adapt https://github.com/alphagov/publishing-e2e-tests/blob/master/spec/support/frontend_helpers.rb#L4 to check app name ends with the correct one (eg by using `$=`)

Reply:
👍 I've updated the docker-compose, the check, and the commit message to explain.

Reply:
Ok great :+1: